kanidm_unix_int/src/daemon.rs

```diff
@@ -393,29 +443,23 @@ async fn main() {
             debug!("Profile -> {}", env!("KANIDM_PROFILE_NAME"));
             debug!("CPU Flags -> {}", env!("KANIDM_CPU_FLAGS"));
 
-            let cfg_path = Path::new("/etc/kanidm/config");
-            let cfg_path_str = match cfg_path.to_str() {
-                Some(cps) => cps,
-                None => {
-                    error!("Unable to turn cfg_path to str");
-                    std::process::exit(1);
-                }
-            };
+            // we can do a blind unwrap because there's a default in clap
```

Comment:
clippy will be sad,  you may need the #[allow(clippy::expect_used)]

Better to use expect in this case than unwrap.

Reply:
Sorted.